openhands/integrations/github/templates/pr_update_prompt.j2

```diff
@@ -0,0 +1,25 @@
+You are working in /workspace, checked out to branch {{ branch_name }}, which has an open PR #{{ pr_number }}.
+A comment on the PR (below) has been addressed to you. Do NOT respond to this comment via the GitHub API. 
+
+{% if file_location %} The comment is on the file `{{ file_location }}`{% endif %}.
```

Comment:
is line number available here?

Reply:
Yeah I can add it in

Reply:
Actually I'll add this information in a separate PR; I'd like to simply move the templates into OSS in this PR